extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -141,11 +145,15 @@ export class DoubleclickA4aEligibility {
     if (useRemoteHtml) {
       warnDeprecation('remote.html');
     }
-    if (hasUSDRD || (useRemoteHtml && !element.getAttribute('rtc-config'))) {
-      return false;
-    }
     let experimentId;
     const urlExperimentId = extractUrlExperimentId(win, element);
+    if (hasUSDRD || (useRemoteHtml &&
+                     !element.getAttribute('rtc-config') &&
+                     (!urlExperimentId ||
```

Comment:
Code golf!  Don't need !urlElementId here as URL_EXPERIMENT_MAPPING[undefined] certainly does not equal remote html experiment.  I'll merge for now but please address later.